src/tasks/command/command_task.c

```diff
@@ -101,6 +101,13 @@ void command_task_dispatch(slate_t *slate)
 
         if (successful_peek)
         {
+            if (!is_packet_authenticated(&packet))
```

Comment:
Suggest you define a build flag so for convenience we can set at build time whether we want to enable packet authentication or not.

In CMakeLists.txt:
option(AUTH "Enable command packet authentication")

Then in here some version of:
#ifdef AUTH

So we can `cmake .. -DAUTH` during build time.

Reply:
fixed in 7bac89e748b56738e42d99497c35feceb72757f7

Reply:
Thanks!